src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -91,6 +93,8 @@ public class OpenLoadFlowParameters extends AbstractExtension<LoadFlowParameters
 
     public static final String TRANSFORMER_VOLTAGE_CONTROL_MODE_NAME = "transformerVoltageControlMode";
 
+    public static final String DC_COS_PHI_NAME = "dcCosPhi";
```

Comment:
@annetill are you sure cos(phi) is the right naming? phi is used in OLF as the voltage angle. Would power factor more usual way of naming this?

Reply:
Yes right, my mistake, I am going to fix the name.